index.html

```diff
@@ -1,13 +1,383 @@
+        leader at Indie Webcamp and Microformats, and a member of the W3C CSS
+        Working Group where he is one of the principal editors of several CSS
+        specifications. Tantek wrote HTML5 Now and shares his thoughts and news
+        on tantek.com. You can find him as @t on Twitter. Thank you for helping
+        shape the web, dear Tantek!
+      </p>
+    </section>
+
+    <section class="shared-links">
+      <div class="link link-1">
+        <div class="avater-box"></div>
+        <div>
+        <h2>Variable Fonts For Developers</h2>
+        <p>
+          Sebastiano Guerriero’s quick tutorial shows how to design a CSS
+          triangle that inherits some properties from its parent thanks to the
+        </p>
+      </div>
+      </div>
+      <div class="link link-2">
+        <div class="avater-box"></div>
+        <div>
+        <h2>The Layout Instability API</h2>
+        <p>
+          Through a series of simple, composable layouts, will teach you how to
+          better harness the built-in algorithms that power browsers and CSS.
+        </p>
+        </div>
+      </div>
+      <div class="link link-3">
+        <div class="avater-box"></div>
+        <div>
+        <h2>Relearn CSS Layout</h2>
+        <p>
+          showcases the many possibilities and opportunities that variable fonts
+          can offer developers and designers on the web.
+        </p>
+      </div>
+    </div>
+      <div class="link link-4">
+        <div class="avater-box"></div>
+        <div>
+        <h2>SuperSearch</h2>
+        <p>
+          Unexpected layout shifts are annoying. The new Layout Instability API
+          helps web developers detect when these shifts are happening to their
+          users. Now available in Chrome behind a feature flag.
+        </p>
+      </div>
+      </div>
+      <div class="link link-5">
+        <div class="avater-box"></div>
+        <div>
+        <h2>National Park Typeface</h2>
+        <p>
+          lets you find the right font for you project based on real optical
+          properties such as weight, x-height, contrast, and proportion.
+        </p>
+      </div>
+      </div>
+      <div class="link link-6">
+        <div class="avater-box"></div>
+        <div>
+        <h2>Rounded Triangles With Clip-Path</h2>
+        <p>
+          The free typeface is designed to mimic the national park service signs
+          carved using a router bit.
+        </p>
+      </div>
+      </div>
+    </section>
+
+    <section class="featured-book">
+      <div class="img-box"></div>
+      <div>
+      <h1>Design systems</h1>
+      <p>
+        Meet Design Systems, a recent Smashing book in which Alla Kholmatova
+        explores how to set up an effective design system to create great
+        digital products. With common traps, gotchas and lessons learned
+      </p>
+      </div>
+      <button href="" role="button" class="btn">Table of Contents&nbsp;↬</button>
```

Comment:
Note that `href` is not allowed on element `button`, kindly do it in the right way.